cmd/metal-api/internal/scaler/poolscaler_test.go

```diff
@@ -0,0 +1,132 @@
+package scaler
+
+import (
+	"testing"
+
+	"github.com/metal-stack/metal-api/cmd/metal-api/internal/metal"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestPoolScaler_AdjustNumberOfWaitingMachines(t *testing.T) {
+	tests := []struct {
+		name      string
+		m         *metal.Machine
+		partition *metal.Partition
+		mockFn    func(mock *MockMachineManager)
+		wantErr   bool
+	}{
+		{
+			name:      "waiting machines match pool size; do nothing",
+			m:         &metal.Machine{SizeID: ""},
```

Comment:
`SizeID` does not have to be defined to it's anyway default value? Even seems like `m` is not at all required for the test.